resolwe/flow/static/flow/processSchema.json

```diff
@@ -56,12 +56,19 @@
         "persistence": {
             "description": "Processor data persistence",
             "type": "string",
-            "enum": ["RAW", "CACHED", "TEMP"]
+            "enum": [
```

Comment:
Move all changes except `"relations"` to a separate commit.

Reply:
Those are from the autoformatter. I'll remove them.

Reply:
Done